langfair/metrics/classification/metrics/false_discovery.py

```diff
@@ -74,10 +74,16 @@ def evaluate(
         cm1 = confusion_matrix(
             y_true[groups == unique_groups[0]], y_pred[groups == unique_groups[0]]
         )
+        print("-------------")
```

Comment:
Can we please remove these print statements or create a verbose option to not show them and set default to not print? Same comment for all child classes. Thanks!

Reply:
Removed the print statements